### ISSUE TITLE
Tests Failing Due to Roundoff

### ISSUE THREAD

**Issue by hphratchian** [MQCPack/mqcPack#52]
A few of the test jobs are failing because of round-off errors. We need to fix this. Ideas?

**Comment by leethomo86** [comment]
The problem is the default print shows numerical noise in some of the output. Perhaps we can add an optional format specifier to printing routines (with the current formatting as the default)?

**Comment by hphratchian** [comment]
That'd be useful. Are these prints all happening with prints that eventually end in MQC_General print routines?

**Comment by leethomo86** [comment]
I don't think so, but it should be easy enough to change. I also recall that the argument order in MQC intrinsic print is inconsistent with MQC algebra1, algebra2 and EST printing. I am not sure that was ever fixed either.

**Comment by leethomo86** [comment]
I added a high precision keyword to some printing routines in algebra 1 as a temporary measure that should be replaced with a more consistent option across all printing routines

**Comment by leethomo86** [comment]
@hphratchian There are no tests failing due to round-off errors to my knowledge. Is this issue now fixed? Can it be closed?
